plugins/tekton/src/components/pipeline-topology/PipelineVisualizationView.tsx

```diff
@@ -63,7 +63,7 @@ export const PipelineVisualizationView = ({
 
   const pipelineRunViz = (
     <WrapperInfoCard
-      allErrors={allErrors}
+      allErrors={undefined}
```

Comment:
Made it `undefined` to hide cluster errors in the PipelineRun visualization

Before:
<img width="1712" alt="Screenshot 2023-07-28 at 11 34 20 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/2e05a7b7-7c42-4041-8e77-8fecde58382a">

After:
<img width="1712" alt="Screenshot 2023-07-28 at 11 31 57 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/9e307846-b680-4def-8bc9-f05fadfd09e3">